packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,17 +1,27 @@
 import React, { createRef } from 'react';
 import * as PopperJS from 'popper.js';
-import { Popper } from './Popper';
-import { PopperController, UsingPopperProps } from './PopperController';
+import { Popover } from './Popover';
+import { PopoverController, UsingPopperProps } from './PopoverController';
 
 export interface TooltipProps extends UsingPopperProps {
   theme?: 'info' | 'error';
 }
+
+export interface PopoverContentProps {
+  updatePopperPosition?: () => void;
+}
+
+export type PopoverContent<T extends PopoverContentProps> =
+  | string
+  | React.ReactElement<T>
+  | ((props: T) => JSX.Element);
```

Comment:
I can't see we ever use this generically typed param. Can you change the props: T int the callback to be typed PopoverContentProps instead of generic. That way we can get rid of all the PopoverContent<any> everwhere.

Reply:
Not sure what you mean by getting rid of it everywhere. Did remove the generic though, so we get rid of the `any`s